WorkingWithText/Program.cs

```diff
@@ -13,14 +13,61 @@ public static class WorkingWithText
         // return bool False.
         public static bool IsConsecutive(string hyphenNum)
         {
-            return default;
+            var splitNum = hyphenNum.Split('-');
+            var numSplitList = new List<int>();
+            int parseVar = 0;
+
+            foreach (var num in splitNum)
+            {
+                if (int.TryParse(num, out parseVar))
+                {
+                    numSplitList.Add(parseVar);
+                }
+                else
+                {
+                    return false;
+                }
+            }
+
+            for (var i = 1; i < numSplitList.Count; i++)
+            {
+                numSplitList.Sort();
+
+                if (numSplitList[i] != numSplitList[i - 1] + 1)
+                {
+                    return false;
+                }
+            }
+            return true;
```

Comment:
You can't just stick return true xD

This will make EVERY test pass. Even if the input is wrong. You just revealed to me that we need to fix our testing.

Reply:
Basically, we should have written a test case for when it improperly descending, right now, the way you wrote it sneaks it by. Thank you for address this.

Reply:
I don' think you ever used line 18?

Keep in mind this is acceptable,

```
            var splitNum = hyphenNum.Split('-');
            var numSplitList = new List<int>();

            foreach (var num in splitNum)
            {
                if (int.TryParse(num, out var parseVar))
                {
                    numSplitList.Add(parseVar);
                }
```

Reply:
This is called in inline try parse declaration